src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -3387,27 +3387,26 @@
      * </pre>
      * <p>parses as:</p>
      * <pre>
-     * --LITERAL_FOR (for)
-     *    |--LPAREN (()
-     *    |--FOR_EACH_CLAUSE
-     *    |   |--VARIABLE_DEF
-     *    |   |   |--MODIFIERS
-     *    |   |   |--TYPE
-     *    |   |   |   `--LITERAL_INT (int)
-     *    |   |   `--IDENT (value)
-     *    |   |--COLON (:)
-     *    |   `--EXPR
-     *    |       `--IDENT (values
-     *    |--RPAREN ())
-     *    `--SLIST ({)
-     *        |--EXPR
-     *        |   `--METHOD_CALL (()
-     *        |       |--IDENT (doSmth)
-     *        |       |--ELIST
-     *        |       `--RPAREN ())
-     *        |--SEMI (;)
-     *        `--RCURLY (})
-     *
+     * LITERAL_FOR -&gt; for
```

Comment:
please add `--` before `LITERAL_FOR` and ident next lines as it was before

Reply:
done